### PR TITLE
Add support for finalized tag.

### PR DESCRIPTION
**Pull request by bgravenorst**
Linea now supports the `finalized` block parameter tag.

### CODE REVIEW

File: docs/developers/reference/api/linea-estimategas.mdx [hunk 1]
```diff
@@ -61,10 +61,13 @@ using `linea_estimateGas` for more accurate results.
   - `value`: _\[optional]_ Hexadecimal value of the value sent with this transaction.
   - `data`: _\[optional]_ Hash of the method signature and encoded parameters. See the
       [Ethereum contract ABI specification](https://docs.soliditylang.org/en/latest/abi-spec.html).
-  - `block number`: _\[optional]_ A string representing a block number, or one of the string tags `latest`, `earliest`, or
-      `pending`. See the [default block parameter](https://ethereum.org/en/developers/docs/apis/json-rpc/#default-block).
-      You can [query the L1 rollup contract](../../guides/finalized-block.mdx) to retrieve the current finalized block.
+  - `block number`: _\[optional]_ A string representing a block number, or one of the string tags `latest`, `earliest`,
+      `pending`, `safe`, or `finalized`.
```
Comment (m4sterbunny) [inline docs/developers/reference/api/linea-estimategas.mdx:65]:
```suggestion
      `pending`, or `finalized`.
```

File: docs/developers/reference/api/linea-getproof.mdx [hunk 2]
```diff
@@ -16,21 +16,17 @@ The supplied block parameter must be an L2 block that has been finalized on L1.
...
-
 ## Parameters
 
 - `address`: A string representing the address (20 bytes) for which the proof is requested.
 - `storageKeys`: An array of 32-byte storage keys to be proofed and included.
-- `blockParameter`: A hexadecimal block number, or one of the string tags `latest`, `earliest`, or
-      `pending`. See the [default block parameter](https://ethereum.org/en/developers/docs/apis/json-rpc/#default-block).
-      You can [query the L1 rollup contract](../../guides/finalized-block.mdx) to retrieve the current finalized block.
+- `blockParameter`: A hexadecimal block number, or one of the string tags  `latest`, `earliest`,
+      `pending`, `safe`, or `finalized`.
```
Comment (m4sterbunny) [inline docs/developers/reference/api/linea-getproof.mdx:24]:
```suggestion
      `pending`, or `finalized`.
```

**Review by m4sterbunny — APPROVED** [review]
LGTM

File: docs/developers/guides/finalized-block.mdx [hunk 3]
```diff
@@ -6,24 +6,34 @@ image: /img/socialCards/retrieve-finalized-l2-blocks.jpg
...
-## Prerequisites
+Use the `finalized` tag in API calls to specify a block that has been confirmed by the L1
+network. For example, here the `eth_getBlockByNumber` method returns information about the current
+finalized block:
+
+```bash
+curl https://rpc.linea.build \
+  -X POST \
+  -H "Content-Type: application/json" \
+  -d '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["finalized",false],"id":1}'
```
Comment (m4sterbunny) [inline docs/developers/guides/finalized-block.mdx:24]:
```suggestion
  -d '{"jsonrpc":"2.0","method":"eth_getBlockByNumber","params":["finalized",false],"id":1}'
```

Comment (m4sterbunny) [reply docs/developers/guides/finalized-block.mdx:24]: This structure ?array? doesn't look right to me, need to verify this

Comment (m4sterbunny) [reply docs/developers/guides/finalized-block.mdx:24]: verified -- tis fine

**Comment by jonesho** [comment]
LGTM

**Review by jonesho — APPROVED** [review]

**Pull request merged**